docs/core/track_fitting.md

```diff
@@ -1,5 +1,36 @@
 # Track Fitting
 
+The track fitting algorithms estimate the track parameters.
+It is part of the pattern recognition/track  reconstruction/tracking.
+We can run the track fitting algorithms, after we allocated all hits to single tracks with the help of a track finding algorithm.
+It is not necessary, that all points of a track are present.
+
+Currently we have implementations for three different fitters:
+* Kalman Filter
+* GSF
+* Global Chi-Square Fitter (GX2F) [wip]
+Even though all of them are least-squares fits, the concepts are quite different.
+Therefore, we should not expect identical results from all of them.
+
+(kf_core)=
+## Kalman Filter (KF) [wip]
+
+:::{note}
```

Comment:
I would remove this and maybe add at the end a `note` box at the end of the chapter that this chapter will be extended in the future.